src/coreclr/scripts/superpmi.py

```diff
@@ -445,6 +457,24 @@ def get_files_from_path(path, matchFunc=lambda path: True):
 
     return files
 
+def run_and_log(command, log_level=logging.DEBUG):
+    """ Return a command and log its output to the debug logger
+
+    Args:
+        command (list) : Command to run
+        log_level (int) : log level to use for logging output (but not the "Invoking" text)
+
+    Returns:
+        Process return code
+    """
+
+    logging.debug("Invoking: " + " ".join(command))
+    proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    stdout_output, stderr_output = proc.communicate()
```

Comment:
nit: I would do `stdout_output, _ = ...` since you are not using `stderr_output `.

Reply:
Thanks; I'm trying to be more "python-ic" over time...